### DISCUSSION TITLE
The input_data of "1.1 -Basics of geological modeling with GemPy"

### DISCUSSION THREAD

**Discussion by YNF0401** [gempy-project/gempy#667] [Q&A] [answered]
Hi:

I am practicing the tutorial of "1.1 -Basics of geological modeling with GemPy" (https://www.gempy.org/tutorials).

In the abovementioned tutorial, it says "In this tutorial, all input data is created by importing such CSV-files. These exemplary files can be found in the input_data folder in the root folder of GemPy".

However, even I searched all the website, I couldn't find the root folder. I wonder maybe it is becuase I am not familiar with how the website functions or whether I should go to somewhere on the GitHub.

Can anyone indicate me where can get these example input files?  Thank you very much.

**Comment by Japhiolite** [comment, marked as answer]
Hi @YNF0401 

thanks for your post. Indeed, we have to rephrase the text in the tutorial. Aiming for a cleaner repository structure, all input data was moved to the repository: https://github.com/cgre-aachen/gempy_data/

In there, you'll find the Input data for Tutorial 1.1 https://github.com/cgre-aachen/gempy_data/tree/master/data/gempy_models/Tutorial_ch1-1_Basics

Best,
Jan

**Reply by YNF0401** [reply]
Hi @Japhiolite 
Thank you very much for your reply.

With the  link you provided, I can now continue my practicing.

Thanks a lot!
